lasagne/updates.py

```diff
@@ -132,8 +138,9 @@ def sgd(loss_or_grads, params, learning_rate):
 
 
 def apply_momentum(updates, params=None, momentum=0.9):
-    """Returns a modified update dictionary that includes momentum terms.
+    """Applies momentum
```

Comment:
Yes. If the table gets too wide it'll get a scrollbar which look terrible. I changed the names slightly, maybe its better now.